configuration/config-file-yaml.md

```diff
@@ -552,4 +552,18 @@ The backslash `\` is interpreted as an escape character. You need `\` for settin
 str_param:   "foo\nbar" # \n is interpreted as actual LF character
 ```
 
+### Parse setting
+You can use `parse:` to set up parse.
+Example: in_tail plugin
+```yaml
```

Comment:
Thanks! Basically looks good!

How about this?

```suggestion
### Parse setting

You can use `parse:` to set up the parser for the input plugin.

Example: in_tail plugin

```yaml
```

Reply:
Before:

![Screenshot from 2024-05-10 13-58-06](https://github.com/fluent/fluentd-docs-gitbook/assets/12229857/8eac9097-dd1a-4025-90df-d3255e2a18f9)

After:

![Screenshot from 2024-05-10 14-04-44](https://github.com/fluent/fluentd-docs-gitbook/assets/12229857/3fab1df4-bf8b-4393-b1e8-fd7c10a4a18f)